make/mosquitto/mosquitto.mk

```diff
@@ -28,6 +28,7 @@ $(PKG)_MAKE_OPTIONS += WITH_DOCS=no
 $(PKG)_MAKE_OPTIONS += WITH_SRV=no
 $(PKG)_MAKE_OPTIONS += WITH_UUID=$(if $(FREETZ_PACKAGE_MOSQUITTO_WITH_UUID),yes,no)
 $(PKG)_MAKE_OPTIONS += WITH_TLS=$(if $(FREETZ_PACKAGE_MOSQUITTO_WITH_SSL),yes,no)
+$(PKG)_MAKE_OPTIONS += WITH_MEMORY_TRACKING=no  # Note: Setting this to "yes" requires uClibc >= 1.0.29
```

Comment:
I modified the poorly worded comment. The heap tracking feature is what needs uClibc >= 1.0.29. The best we currently have is 1.0.14, which does not provide the needed malloc_usable_size() call.

Reply:
Ok, got it. My fault, I've read 1.0.29 as 0.9.29 and got confused.